docs/api.rst

```diff
@@ -1227,6 +1227,58 @@ It's also possible to embed `Materialized Views <https://www.postgresql.org/docs
 
   If view definitions change you must refresh PostgREST's schema cache for this to work properly. See the section :ref:`schema_reloading`.
 
+Embedding Views Containing Joins
+--------------------------------
+
+The same embedding rules applies to views with joined tables, that is, the column of any table participating in the join with a **foreign key** definition must be present in the view.
+
+For instance, let's create a view called ``film_competition_view`` that will join three tables:
+
+.. code-block:: postgres
+
+  CREATE VIEW film_competition_view AS
+  SELECT
+   films.title as film_title
+   , competitions.name as competition_name
+   , film_id
+  FROM
+   nominations
+   JOIN films ON films.id = nominations.film_id
+   JOIN competitions ON competitions.id = nominations.competition_id;
+
+The view column can embed the table ``films`` because it has the ``film_id`` column that has a foreign key defined.
+
+But what if you want to embed the view with the ``actors`` table? The ``film_id`` column alone will not allow it since it belongs to ``nominations`` and there is no relationship between that table and ``actors`` directly. In this case, you need to add the ``film.id`` column which does have the foreign key to the ``actors`` table.
```

Comment:
> ``film.id`` column which does have the foreign key to the ``actors`` table.

The film doesn't have an FK to the `actors` table right? Only an m2m relationship through `roles`(which has the FK)

Reply:
True, I clarified the explanation now.